svid/jwtsvid/tests/check_svid.c

```diff
@@ -310,6 +236,62 @@ START_TEST(test_jwtsvid_error5)
 }
 END_TEST
 
+// precondition: invalid jwt token with subject not spifeeid
```

Comment:
I would recommend to add infortation about what is expected in this test (postcondition).
Like an error or success